build.gradle

```diff
@@ -47,3 +43,11 @@ subprojects {
         }
     }
 }
+
+
+def getSecondDotPosition(String version) {
+    return version.split(/\./)[0].length() + 1 + version.split(/\./)[1].length()
+}
+def getTotalNumbersAfterDash(String version) {
```

Comment:
This is not even used (well... doubly defined... again in apix-docker.build.gradle). Delete.

Reply:
removed methods